src/coreclr/jit/fgbasic.cpp

```diff
@@ -579,7 +579,6 @@ void Compiler::fgReplaceJumpTarget(BasicBlock* block, BasicBlock* newTarget, Bas
                 if (jumpTab[i] == oldTarget)
                 {
                     jumpTab[i] = newTarget;
-                    break;
```

Comment:
AFAICT, this is safe and won't cause diffs. However, the header comment specifically says:
```
// 2. Only the first target found is updated. If there are multiple ways for a block
//    to reach 'oldTarget' (e.g., multiple arms of a switch), only the first one found is changed.
```
so that should be updated.

One caller, fgNormalizeEHCase2() specifically expects the old behavior:
```
// Now change the branch. If it was a BBJ_NONE fall-through to the top block, this will
// do nothing. Since cheap preds contains dups (for switch duplicates), we will call
// this once per dup.
```
but it's ok, because subsequent calls will just do nothing.

Unrelated, I also note the comment says:
```
 4. The switch table "unique successor" cache is invalidated.
```
Although we don't call `InvalidateUniqueSwitchSuccMap()`, so that comment is a little misleading.

Reply:
Thanks, let me update this documentation.

Reply:
Added the invalidation and updated the comments.